target/linux/rockchip/patches-5.15/105-nanopi-r4s-sd-signalling.patch

```diff
@@ -10,6 +10,16 @@ userspace or following a kernel panic is always working.
 
 Signed-off-by: David Bauer <mail@david-bauer.net>
 
+--- a/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
++++ b/arch/arm64/boot/dts/rockchip/rk3328-roc-cc.dts
+@@ -335,7 +335,6 @@
+ 	sd-uhs-sdr12;
+ 	sd-uhs-sdr25;
+ 	sd-uhs-sdr50;
+-	sd-uhs-sdr104;
```

Comment:
Why do you remove this line?
Please explain in the commit message why you do these changes to the dts files.

Reply:
the rockchip boards have a common bug when operating uhs speed, which will hang the system during a soft reboot.

Reply:
Please add this into a separate commit with the explanation you gave in this comment in the commit comment.

Reply:
done ;)